src/StackHCI/custom/stackhci.ps1

```diff
@@ -234,18 +237,33 @@ $ClusterScheduledTaskReadyState = "Ready"
 
 $GetArcSettingsWaitTimeMinutes = 1
 $GetArcSettingsSleepTimeSeconds = 15
-$ArcSettingsVerificationLimit = 5
+$ArcSettingsVerificationLimit = 10
 
 $ArcSettingsDisableInProgressState = "DisableInProgress"
 
+$defaultLogsDirectory = "C:/ProgramData/AzureStackHCI/Registration"
```

Comment:
Please use `System.Environment.SpecialFolder.CommonApplicationData` instead of OS hardcoded path. [Here](https://learn.microsoft.com/en-us/dotnet/api/system.environment.specialfolder?view=netframework-4.7.2) is more info for this.

Reply:
Great, thank you for this. I've updated the script.